examples/mwe/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
I would remove the `examples/mwe` directory entirely, since it seems to be related to testing only